p751toolbox/field_amd64.s

```diff
@@ -1417,7 +1417,370 @@ TEXT ·fp751Mul(SB), $96-24
 
 	RET
 
-TEXT ·fp751MontgomeryReduce(SB), $0-16
+#define mul256x448bmi2adx(M0, M1, C, T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10) \
```

Comment:
Comments are missing - here and below. What M0, M1, C, .... stand for? Where is input? Where is output? Macro makes assumptions about base pointer. This all needs to be documented in production ready version of the code